docs/clients/nft-client.md

```
@@ -331,6 +333,42 @@ long totalSupply = nftClient.wipeNfts(
 
 ---
 
+## Pause NFT
+
+Pauses an NFT type, preventing NFT transfers and other token operations while the type is paused. The pause key must sign the transaction. The NFT type must have a pause key configured.
+
+```java nftClient.pauseNft(tokenId);
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)